src/main/java/frc/robot/subsystems/PhotonVision.java

```diff
@@ -140,7 +140,8 @@ public void periodic() {
         }
 
         LightningShuffleboard.setBool("Vision", "HasResult", result.hasTargets());
-        LightningShuffleboard.set("Vision", "Timestamp", result.getTimestampSeconds());
+        LightningShuffleboard.set("Vision", "timestamp", result.getTimestampSeconds());
+        LightningShuffleboard.set("Vision", "Status Codes", lastEstimatedRobotPose);
```

Comment:
uh... no
just delete this one for now, the point of this was to ensure we were getting a consistent camera feed and the pi/limelight were both working right